msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -336,6 +336,23 @@ The query parameters are:
 > [!NOTE]
 > You can see `channelId` and `groupId` in the URL from the channel.
 
+### Generate deep links to chat messages
+
+Use this deep link format to go to a message in personal and group chats in Teams:
+
+`http://teams.microsoft.com/l/message/{chatId}/{messageId}?context={"contextType":"chat"}`
+
+Example: `http://teams.microsoft.com/l/message/19:253f5895-9a62-4362-8d38-43f0205c702c_f1b94dcf-0aa3-4989-bcdf-ef4a5ed00f86@unq.gbl.spaces/1563480968434?context=%7B%22contextType%22:%22chat%22%7D`
+
+The query parameters are:
+
+* `chatId`: ChatId of the conversation. You can get the `chatId` from URL of the chat in Teams web client or using the [Microsoft Graph API](/graph/api/chat-get?view=graph-rest-1.0&tabs=http&preserve-view=true). For example, `19:253f5895-9a62-4362-8d38-43f0205c702c_f1b94dcf-0aa3-4989-bcdf-ef4a5ed00f86@unq.gbl.spaces`.
+
+  Apps can read a group chat ID through context using the Microsoft Teams JavaScript library or from the incoming request payload to Bot.
+
+* `messageId`: Message ID of the conversation. For example, `1563480968434`.
+* `contextObject`: Specify the contextType as chat.
```

Comment:
The query parameter name is "context", not "contextObject"

Reply:
updated the parameter.